packages/meditrak-app/app/assessment/actions/submitSurvey.js

```diff
@@ -98,7 +98,7 @@ const processQuestions = async (dispatch, getState, database, userId, questions)
   const answers = getAnswers(getState());
   for (const question of questions) {
     const answer = answers[question.id];
-    if (answer === undefined) {
+    if (answer === undefined || answer === null || answer === '') {
```

Comment:
This fixes the original issue I introduced in the hot fix: answers that were previously filtered out are now making it through. Still allows the 0's to successfully make it through, which was the motivation for the original hot fix.